hyper/hyper/resume_test.go

```diff
@@ -0,0 +1,60 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"libvirt.org/libvirt-go"
+)
+
+func TestResume(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) *libvirt.Domain
+		ExpectedErr    string
+	}{
+		"resume the desktop correctly": {
```

Comment:
It should check the state of the desktop afterwards

Reply:
I don't get the point here as it is done afterwards during the test (I assume)

Reply:
For example: 

1- Create a running desktop
2- Pause & Resume the desktop
3- Check that the desktop is running